Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -141,6 +154,21 @@ export default class ReportsView extends Vue {
                             </div>
                         </b-col>
                     </div>
+                    <div
+                        v-else-if="reportType == 'COVID-19'"
+                        data-testid="covid19ReportSample"
+                    >
+                        <b-col>
```

Comment:
this changed a bit too, look at the same PR https://github.com/bcgov/healthgateway/pull/1425/files

Reply:
updated